media-libs/libmpdclient/libmpdclient-2.14.recipe

```diff
@@ -41,6 +44,9 @@ BUILD_PREREQUIRES="
 	cmd:ninja
 	"
 
+defineDebugInfoPackage libmpdclient$secondaryArchSuffix \
+	$libDir/libmpdclient.so.$libVersion
+
 BUILD()
 {
 	rm -rf build_haiku || true
```

Comment:
```diff
-	rm -rf build_haiku || true
+	rm -rf build_haiku
```
(The `|| true` is unneeded because `rm` always returns 0 as exit code if `-f` is passed.)